src/components/Book.js

```diff
@@ -1,16 +1,46 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
 
 const Book = ({ book, handleRemoveBook }) => {
   const handleBookRemove = () => {
     handleRemoveBook(book);
   };
+  const percentage = 66;
   return (
-    <tr key={book.id} id={book.id}>
-      <td>{book.title}</td>
-      <td>{book.category}</td>
-      <td><button type="button" onClick={handleBookRemove} className="btn btn-danger">Delete</button></td>
-    </tr>
+    <div className="card">
+      <div className="row card-inside">
+        <div className="col-12 col-sm-3 col-md-4">
+          <p className="gender-book">{book.category}</p>
+          <p className="title-book">{book.title}</p>
+          <p className="author-book mb-2">Suzanne Collins</p>
+          <div className="row">
+            <div className="col-12 col-lg-4 co-first">
+              <a className="book-option" href="/">Comments</a>
+            </div>
+            <div className="col-12 col-lg-3 co-middle">
+              <a className="book-option" href="/">Remove</a>
+            </div>
+            <div className="col-12 col-lg-3 co-last">
+              <a className="book-option" href="/">Edit</a>
+            </div>
+          </div>
+        </div>
+        <div className="col-4 col-sm-3 col-md-2 col-lg-1 mt-2">
+          <CircularProgressbar value={percentage} />
+        </div>
+        <div className="col-12 col-sm-3 col-md-2">
+          <h2>28%</h2>
+          <p className="perc-completed">Completed</p>
+        </div>
+        <div className="col-12 col-sm-3 col-md-4 card-last-div">
+          <p className="current-chapter">Current chapter</p>
+          <p>Chapter 13</p>
+          <button type="button" onClick={handleBookRemove} className="btn btn-update">Update progress</button>
+        </div>
+      </div>
```

Comment:
## Required change
In line with what @Elbie-em pointed out, I think he meant the functionality of the `remove` button is misleading. 
- [ ] You might want to replace the delete functionality that gets conducted in the `Update Progress` button and move it to the `Remove` button. That should be an easy fix owing to the load of work you have poured into this already. The `Update Progress` button can be blank for now.

Kindly do this one last change and submit it for review. 

That should be all.

Reply:
Done, I'll push changes.
![image](https://user-images.githubusercontent.com/20563252/125970595-b7b53d6d-a51e-46cc-8a33-d57994667055.png)